cheroot/workers/threadpool.py

```diff
@@ -170,6 +174,29 @@ def __init__(
         self._queue_put_timeout = accepted_queue_timeout
         self.get = self._queue.get
         self._pending_shutdowns = collections.deque()
+        self.log = logger or (lambda msg: None)
+        self.monitor = None
+        self.monitor_configured = False
+
+    def configure_monitor(
+        self,
```

Comment:
pylint will be unhappy with this.

Reply:
Can you please elaborate? any specific thing/pylint error to look out for? When I run pylint on current code there seems to be lot of W,C,R issues to be present.